201-logic-app-veter-pipeline/README.md

```diff
@@ -0,0 +1,38 @@
+# Azure Logic Apps - VETER Pipeline
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-logic-app-veter-pipeline%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-logic-app-veter-pipeline%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template creates a VETER pipeline using Logic Apps. It creates an integration account, adds schema/map into it, creates a logic app and associates it with the integration account. The logic app implements a VETER pipeline using Xml Validation, XPath Extract and Transform Xml operations.
+`Tags: VETER, Logic Apps, Integration Account, Enterprise Integration`
+
+## Prerequisites ##
+
+In order to properly deploy this ARM template, you need to first create a function app/container and provide the name of that function app in the parameters.
+Create an Azure Functions container - https://portal.azure.com/#create/Microsoft.FunctionApp
```

Comment:
We need more detail here - I followed the link but it wasn't sufficient to get the template to deploy.  What else is needed and how do I do it?